doc/robot_model_and_robot_state/src/robot_model_and_robot_state_tutorial.cpp

```diff
@@ -34,18 +34,31 @@
 
 /* Author: Sachin Chitta, Michael Lautman*/
 
-#include <ros/ros.h>
-
 // MoveIt
 #include <moveit/robot_model_loader/robot_model_loader.h>
 #include <moveit/robot_model/robot_model.h>
 #include <moveit/robot_state/robot_state.h>
 
+// All source files that use ROS logging should define a file-specific
+// static const rclcpp::Logger named LOGGER, located at the top of the file
+// and inside the namespace with the narrowest scope (if there is one)
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("robot_model_and_robot_state_demo");
+
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "robot_model_and_robot_state_tutorial");
-  ros::AsyncSpinner spinner(1);
-  spinner.start();
+  rclcpp::init(argc, argv);
+  rclcpp::NodeOptions node_options;
+
+  // This enables loading undeclared parameters
+  // best practice would be to declare parameters in the corresponding classes
+  // and provide descriptions about expected use
+  node_options.automatically_declare_parameters_from_overrides(true);
+  auto robot_node = rclcpp::Node::make_shared("robot_model_and_robot_state_tutorial", node_options);
+
+  // This particular tutorial doesn't require a spinner
+  //  rclcpp::executors::SingleThreadedExecutor executor;
+  //  executor.add_node(robot_node);
+  //  std::thread([&executor]() { executor.spin(); }).detach();
```

Comment:
Go ahead and delete these lines, in my opinion

Reply:
Ok